bcda/lambda/cclf/main_test.go

```diff
@@ -14,15 +14,15 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-type CclfImportMainSuite struct {
+type AttributionImportMainSuite struct {
```

Comment:
I like that we are changing these names.  Do you think it would cause a lot of trouble to also rename the package and directory name to attribution as well?  Would we have to update terraform?

Reply:
I'm not sure about updating the terraform, but we'd have to update the other workflows associated with cclf.

Reply:
Yea, we can do that if it conflicts with your changes.